src/models/WorkerPoolResponse.ts

```diff
@@ -0,0 +1,13 @@
+import { DateString } from '@/types/dates'
+
+export type WorkerPoolResponse = {
+  id: string,
+  created: DateString,
+  updated: DateString,
+  name: string,
+  description: string | null,
+  type: string |null,
```

Comment:
```suggestion
  type: string | null,
```
@stackoverfloweth maybe we can lint this as well?

Reply:
https://github.com/PrefectHQ/eslint-config/issues/20